e2e/cypress/integration/markdown/markdown_image_spec.js

```diff
@@ -22,8 +22,8 @@ describe('Markdown', () => {
 
     const baseUrl = Cypress.config('baseUrl');
 
-    const inlineImage1 = `<h3 class="markdown__heading">In-line Images</h3><p>Mattermost/platform build status:  <a class="theme markdown__link" href="https://travis-ci.org/mattermost/platform" rel="noreferrer" target="_blank"><button class="style--none" alt="Build Status" aria-label="file thumbnail"><img class="markdown-inline-img" alt="image placeholder" src="${baseUrl}/api/v4/image?url=https%3A%2F%2Ftravis-ci.org%2Fmattermost%2Fplatform.svg%3Fbranch%3Dmaster"></button></a></p>`;
-    const inlineImage2 = `<h3 class="markdown__heading">In-line Images</h3><p>GitHub favicon:  <button class="style--none" alt="Github" aria-label="file thumbnail"><img class="markdown-inline-img" alt="image placeholder" src="${baseUrl}/api/v4/image?url=https%3A%2F%2Fgithub.githubassets.com%2Ffavicon.ico"></button></p>`;
+    const inlineImage1 = `<h3 class="markdown__heading">In-line Images</h3><p>Mattermost/platform build status:  <a class="theme markdown__link" href="https://travis-ci.org/mattermost/platform" rel="noreferrer" target="_blank"><div class="style--none file-preview__button"><img class="markdown-inline-img" alt="Build Status" aria-label="file thumbnail" tabindex="0" src="${baseUrl}/api/v4/image?url=https%3A%2F%2Ftravis-ci.org%2Fmattermost%2Fplatform.svg%3Fbranch%3Dmaster"></div></a></p>`;
```

Comment:
Just a suggestion. Since the format is prone to change in the future, just like how it did this time, instead of validating the entire html, can we just validate them separately?

Reply:
I'm late to the party (merged already) but will consider this in the future.  Thanks